packages/pirateship/src/screens/Shop.tsx

```diff
@@ -112,8 +113,8 @@ const ShopStyle = StyleSheet.create({
 
 export interface ShopProps
```

Comment:
Since there is no additional props, this can be a type. 
`export type ShopProps = ScreenProps & Pick<........>`

Reply:
add to punch list